website/docs/plugins/plugin-tezos-client.mdx

```diff
@@ -0,0 +1,110 @@
+---
+
+The Tezos Client plugin provides tasks for interacting with the Tezos client including type checking of Michelson smart contracts, and simulating smart contracts
+
+In future releases, this plugin will be extended to support more Tezos Client features and interacting with testnets
+
+## Requirements
+
+- Node.js v16 or later
+- Docker v0.8.2 or later
+- Taqueria v0.0.6 or later
+- [Taqueria Flextesa plugin](../plugin-flextesa) v0.2.1 or later
+
+:::note
+Currently, the Tezos Client plugin requires the Tezos Client to be running in a local Flextesa sandbox. The recommended approach is to install the Taqueria Flextesa plugin on the project, and then run `taq start sandbox local` to start up the sandbox so the Tezos Client plugin can connect to it
+:::
+
+## Installation
+
+To install the Tezos Client plugin on a Taqueria project, navigate to the project folder and run:
+```shell
+taq install @taqueria/plugin-tezos-client
+```
+
+## Configuration
+
+No project level configuration is available for the Tezos Client plugin
+
+## Taqueria Tasks
+
+The Tezos Client plugin currently provides the following tasks:
+1. `taq typecheck <inputFile>` - typechecks one or more Michelson smart contracts
+2. `taq simulate [inputFile] [inputData]` - simulates calling a Michelson smart contract
+
+As this plugin calls the tezos-client instance that is running in the sandbox, the protocol that will be used in the typecheck or simulation is the one that is currently running in the sandbox which can be configured in `./.taq/config.json`
+
+### The `taq typecheck` task
+
+The `taq typecheck` task typechecks one or more Michelson smart contracts against the Tezos network running in the Flextesa sandbox
+
+The structure of the command for the typecheck task including all optional parameters is:
+```shell
+taq typecheck <inputFile> -s sandboxName
+```
+
+A particular Michelson smart contract can be specified by the optional `<inputFile>` argument. If the file name is not specified, the task will typecheck all Michelson smart contracts in the `/artifacts` and `/contracts` directories
+
+### Optional Parameters
+
+The `typecheck` task will accept the following optional parameters:
+
+| flag  |  name       | description                           |   
+|:-----:|:------------|---------------------------------------|
+|  -s   | sandboxName | The name of the sandbox to target.    |
+
+### The `taq simulate` task
+
+The `taq simulate` task uses the tezos-client instance running in the Flextesa sandbox to simulate calling a Michelson smart contract. You pass in the input parameters for the contract's `main` entrypoint, and specify the storage value for the contract through an optional flag or via the initial storage value set for the contract in the sandbox property of `config.json` 
+
+The structure of the command for the typecheck task including all optional parameters is:
+```shell
+taq simulate [inputFile] [inputData] -s sandboxName --storage valueOfInitialStorage
```

Comment:
Could we change it to `<inputFile> <inputData>`? Angle brackets signify mandatory arguments

Reply:
Yes, my bad on that.... keep getting it backwards